ports/crashpad/vcpkg.json

```
@@ -1,14 +1,18 @@
 {
   "name": "crashpad",
-  "version-date": "2020-03-18",
-  "port-version": 2,
+  "version-date": "2022-04-16",
```

```suggestion
  "version-date": "2022-04-12",
```
Please use the actual commit date

---

> Please use the actual commit date

Wait. I'm not convinced this is the de-facto policy.
(I don't see it written, except for [old hints about `version-string`](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/manifest-files.md#version-string):
`we use the date that the commit was accessed by you`)

---

I saw several reviews where it was asked to use commit date as version-date. However, after some research I saw both policies.